spec/text-editor-component-spec.js

```diff
@@ -4028,9 +4028,9 @@ describe('TextEditorComponent', function () {
       component.setFontSize(10)
       await nextViewUpdatePromise()
       expect(editor.getDefaultCharWidth()).toBeCloseTo(6, 0)
-      expect(editor.getKoreanCharWidth()).toBeCloseTo(9, 0)
+      expect(editor.getKoreanCharWidth()).toBeCloseTo(10, 0)
       expect(editor.getDoubleWidthCharWidth()).toBe(10)
-      expect(editor.getHalfWidthCharWidth()).toBe(5)
+      expect(editor.getHalfWidthCharWidth()).toBeCloseTo(6, 5)
```

Comment:
I took a 👀 at https://travis-ci.org/atom/atom/jobs/123412388#L291-L303, and it seems like these specs are failing. What do you think about changing them to look similar to the solution proposed above?